players.md

```diff
@@ -0,0 +1,85 @@
+| **DTS**                 | ✅                                                                                                             | ❌                                                                           |
+| **EAC3**                | ✅                                                                                                             | ✅                                                                           |
+| **FLAC**                | ✅                                                                                                             | ✅                                                                           |
+| **MP1, MP2, MP3**       | ✅                                                                                                             | ✅ (MP3 only)                                                                |
+| **Opus**                | ✅                                                                                                             | ❌                                                                           |
+| **PCM**                 | ✅                                                                                                             | ✅ Limited support                                                           |
+| **Speex**               | ✅                                                                                                             | ❌                                                                           |
+| **Vorbis**              | ✅                                                                                                             | ❌                                                                           |
+| **WavPack**             | ✅                                                                                                             | ❌                                                                           |
+| **WMA (Lossless, Pro)** | ✅                                                                                                             | ❌                                                                           |
+
+---
+
+## Video Support
+
+| Video Codec             | Swiftfin (VLCKit)                                                                                               | Native (AVKit)                                                               |
+|-------------------------|-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|
+| **AV1**                 | ✅                                                                                                             | ❌                                                                           |
```

Comment:
Note for future research: this might have changed for Native since iOS/tvOS 18. There were some announcements/updates to Apple's HLS specs at WWDC 24, and AV1 with Dolby Vision profile 10 seems supported now, but I haven't been able to modify device profile and do any tests yet.